docs/tutorials/q32d14.ipynb

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
Please format notebook with [nbfmt](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/):

```
$ python3 -m pip install -U --user git+https://github.com/tensorflow/docs

$ python3 -m tensorflow_docs.tools.nbfmt notebook.ipynb
```

Reply:
I did that.